### PR TITLE
Modernize generate files

### PR DESCRIPTION
**Pull request by swamper123**
Pep8 conformity
Remove obsolete generate script (the one with .bsd files)
Reduce write instructions by merging them together
Remove unused vars
Replaced prints with logs

### CODE REVIEW

File: schemas/generate_model.py [hunk 1]
```diff
@@ -24,31 +20,18 @@
...
-
-# "NodesToRead": "AttributesToRead"} # "MonitoringMode": "Mode",
-# "NotificationMessage": "Notification", "NodeIdType": "Type"}
-
-
-# some object are defined in extensionobjects in spec but seems not to be in reality
-# in addition to this list all request and response and descriptions will not inherit
-# NoInherit = ["RequestHeader", "ResponseHeader", "ChannelSecurityToken", "UserTokenPolicy", "SignatureData",
-# "BrowseResult", "ReadValueId", "WriteValue", "BrowsePath", "BrowsePathTarget", "RelativePath",
-# "RelativePathElement", "BrowsePathResult"]#, "ApplicationDescription", "EndpointDescription"
```
Comment (oroulet) [inline schemas/generate_model.py:43]: I did not notice these changes, but they are highly suspicious. They have a huge impact on the low level code

File: schemas/generate_model.py [hunk 2]
```diff
@@ -24,31 +20,18 @@
...
+OverrideTypes = {}
 
 
 class Bit(object):
```
Comment (oroulet) [inline schemas/generate_model.py:29]: if you want to clena up things. These should be dataclasses

File: schemas/generate_model.py [hunk 3]
```diff
@@ -339,32 +300,17 @@ def parse(self):
...
                 self.model.enum_list.append(enum.name)
-            # else:
-            # print("Not implemented node type: " + tag + "\n")
+            else:
+                _logger.debug("Not implemented node type: " + tag + "\n")
         return self.model
 
     def add_extension_object(self):
         obj = Struct()
-        obj.name = 'ExtensionObject'
```
Comment (oroulet) [inline schemas/generate_model.py:348]: the deleted extensionobject is probably the one giving me issues here...

File: schemas/generate_model.py (hunk 1 above)
Comment (swamper123) [reply schemas/generate_model.py:43]: We use out commented code on lowlevel?

File: schemas/generate_model.py (hunk 3 above)
Comment (swamper123) [reply schemas/generate_model.py:348]: I agree that this shouldn't be. Just the fields should be compromised.

**Pull request merged**